pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -6,9 +6,11 @@ import (
 	"encoding/json"
 	"fmt"
 	"reflect"
+	"sort"
 	"strings"
 	"time"
 
+	yaml "gopkg.in/yaml.v2"
```

Comment:
@stlaz this was the culprit of yaml issues. I've fixed that and changed configmap merging to `MergePrunedConfigMap` in #285.